internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/ControllerClient.java

```diff
@@ -994,19 +995,30 @@ public VersionResponse getStoreLargestUsedVersion(String clusterName, String sto
     return request(ControllerRoute.GET_STORE_LARGEST_USED_VERSION, params, VersionResponse.class);
   }
 
-  public RegionPushDetailsResponse getRegionPushDetails(String storeName, String clusterName) {
-    QueryParams params = newParams().add(CLUSTER, clusterName).add(NAME, storeName);
+  public RegionPushDetailsResponse getRegionPushDetails(
+      String storeName,
+      String clusterName,
+      boolean isPartitionDetailEnabled) {
+    QueryParams params = newParams().add(CLUSTER, clusterName)
+        .add(NAME, storeName)
+        .add(PARTITION_DETAIL_ENABLED, isPartitionDetailEnabled);
     return request(ControllerRoute.GET_REGION_PUSH_DETAILS, params, RegionPushDetailsResponse.class);
   }
 
-  public StoreHealthAuditResponse listStorePushInfo(String clusterName, String parentControllerUrl, String storeName) {
-    QueryParams params = newParams().add(CLUSTER, clusterName).add(NAME, storeName);
+  public StoreHealthAuditResponse listStorePushInfo(
+      String clusterName,
+      String parentControllerUrl,
```

Comment:
Are these two parameters needed at all? Probably not. Because ControllerClient already knows which cluster it points to and knows the url. `add(CLUSTER, clusterName)` is not needed either since newParams -> addCommonParams already adds it. So the method could be simplified to:
```
public StoreHealthAuditResponse listStorePushInfo(String storeName, boolean isPartitionDetailEnabled) {
    QueryParams params = newParams().add(NAME, storeName).add(PARTITION_DETAIL_ENABLED, isPartitionDetailEnabled);
    return request(ControllerRoute.LIST_STORE_PUSH_INFO, params, StoreHealthAuditResponse.class);
  }
```

Reply:
It makes sense to me. Will fix it.